code/modules/reagents/reagent_dispenser.dm

```diff
@@ -155,6 +155,8 @@
  * Other dispensers will scatter their contents within range.
  */
 /obj/structure/reagent_dispensers/proc/boom()
+	if(QDELETED(src))
+		return // little bit of sanity sauce before we wreck ourselves somehow
```

Comment:
i could take or leave this being here but we're primed for collision with a whole buncha different stuff acting on each other with how zany the game code is now